src/Tempest/Console/Commands/DiscoveryClearCommand.php

```diff
@@ -32,6 +32,10 @@ public function __invoke(): void
 
             $discovery->destroyCache();
 
+            if ($this->appConfig->environment->isTesting()) {
```

Comment:
This is actually needed during testing, however, because we call it during testing bootstrap, it only triggers once per test run